bms-monitor.js

```diff
@@ -1,18 +1,30 @@
-const {expect} = require('chai');
+const {TemperatureRange, SOCRange, ChargeRateRange}=require("./inputParameterRanges.js")
 
-function batteryIsOk(temperature, soc, charge_rate) {
-    if (temperature < 0 || temperature > 45) {
-        console.log('Temperature is out of range!');
+//checks whether inputParameters are in range or not and specifies the type of breach that is low or high
+function checkBreach(inputParameter, value, min, max) {

```

Comment:
checkBreach returns a boolean, but when is it `true`? To determine that, I need to look into its implementation.
Try making it clear in its name itself. Use the 'is' form. Like `isInRange` - then it's clear that `true` means it is within the min and max